docker/BuildJekyll.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:3.0.0
+
+WORKDIR /srv/jekyll
+COPY ./static/Gemfile .
+
+ARG uid
+ARG uname
+ARG gid
+ARG gname
+
+RUN groupadd -g $gid $gname
+RUN useradd -ms /bin/bash -u $uid -g $gid $uname
+
+RUN gem install jekyll bundler
+RUN bundle install
+
+USER 1000:1000
+ENV GEM_HOME=/usr/local/bundle
+ENV PATH="/usr/local/bundle/bin:${PATH}"
```

Comment:
I don't know a lot about Ruby stuff, but this feels like it should come before the gem & bundle installs? I mean, maybe it's OK because it's describing what bundler already did, it just feels weird to me to set it last.

Reply:
This is for the _run_ portion, where I"m running the container running jekyll. Although I'm wondering now if that stays around in the environment like I was thinking it was?

Reply:
It does, yeah: ENV directives get baked in to the image and all subsequent commands (including a `docker run`) "see" them.

'member how I said that docker _allows_ you to have nice fast reproducible builds, but doesn't really _help_ you do that? This is that lack of support being realized, and I'm not able to act as a stand-in oracle for how the tool ought to be guiding you because I don't know Ruby very well.

What I do know is that `GEM_HOME` feels like "hey you should look for & install gems here", and so it just feels a little weird to say that _after_ installing the gems, y'know? Like, if it works, it works—and maybe it wouldn't if you moved these before the `gem install` lines, idk—but this feels like a thing I'd usually resolve by going and looking at what other Ruby practitioners are calling "best practices" or w/e for their Dockerfiles.